pemFioi/quickAlgo/i18n.js

```diff
@@ -67,6 +72,12 @@ var quickAlgoLanguageStrings = {
       descriptionEdition: "Description :",
       saveAndQuit: "Sauvegarder & Quitter",
       quitWithoutSavingConfirmation: "Quitter sans sauvegarder vos modifications ?",
+      about: "À propos",
+      license: "license :",
```

Comment:
```suggestion
      license: "License :",
```

Same for translations.

Reply:
Added uppercase letters where needed